src/Sylius/Bundle/RbacBundle/spec/Provider/SecurityIdentityProviderSpec.php

```diff
@@ -52,6 +53,13 @@ function it_returns_null_if_token_exists_but_still_no_authenticated_user($securi
         $this->getIdentity()->shouldReturn(null);
     }
 
+    function it_returns_null_if_token_exists_but_its_an_anonymous_user($securityContext, AnonymousToken $token)
+    {
+        $securityContext->getToken()->shouldBeCalled()->willReturn($token);
```

Comment:
ok, it is not really important :), @aramalipoor please revert your last change and we will merge it :)